template-params.yaml

```diff
@@ -0,0 +1,55 @@
+### Data type
+data_type: "fastq"
+data_type: "bam"
```

Comment:
Let's be more explicit here, maybe `input_type` or `input_data_type`

Reply:
Probably we should comment out the one that we're not using by default, so the template is itself useable